texroot/tex/latex/GEWISAgenda/GEWISAgenda.cls

```diff
@@ -1,5 +1,6 @@
-%% GEWIS Agenda Tex style v1.0
+%% GEWIS Agenda Tex style v1.1
 %% Stijl - Gijs de Man, Rink Pieters
+%% Minor textual changes made in 2025 by Jens Knapen
```

Comment:
I think it might be better to remove the author notice and move it to a separate file in the repo? Since this is also not the place for a changelog imho 

(Not that I am advocating that you should not get any recognition)